lib/generators/flipper/templates/migration.rb

```diff
@@ -12,12 +12,10 @@ def self.up
       t.string :value
       t.timestamps null: false
     end
-    add_foreign_key :flipper_gates, :flipper_features, on_delete: :cascade
```

Comment:
I'm hesitant to diverge this fork from the main installation here.

Foreigner is what adds that command to the 3.2 installation I was using. How was it failing to load for you?

Maybe these contributions make most sense back on the main https://github.com/bgentry/flipper-activerecord repo which I believe has support for Rails 4.0 today? This particular repo was setup to ensure compatibility with Rails 3.2 (hence the name).